installer/datafiles/base_container.data

```diff
@@ -36,25 +36,27 @@ MAINTAINER:              'Microsoft Corporation'
 /var;                                                   755; root; root; sysdir
 /var/opt;                                               755; root; root; sysdir
 
-/etc/opt/microsoft;                                     755; root; root
+/etc/opt/microsoft;                                     755; root; root; sysdir
 /etc/opt/microsoft/docker-cimprov;                      755; root; root
 /etc/opt/microsoft/docker-cimprov/conf;                 755; root; root
 
-/etc/opt/omi;                                           755; root; root
-/etc/opt/omi/conf;                                      755; root; root
-/etc/opt/omi/conf/omiregister;                          755; root; root
+/etc/opt/omi;                                           755; root; root; sysdir
+/etc/opt/omi/conf;                                      755; root; root; sysdir
+/etc/opt/omi/conf/omiregister;                          755; root; root; sysdir
 /etc/opt/omi/conf/omiregister/root-cimv2;               755; root; root
```

Comment:
Great question, one I had to research. The answer is: No, this should not be 'sysdir'. It is not created by OMI, so I guess Docker will need to create the directory.

I thought it was a "red flag" when Docker used this for the registration files, but then there were enough other issues where I didn't raise it. As long as nobody else uses root/cimv2, though, we'll be fine. Or we'll need to change this and then change the OMS configuration for a new namespace down the road. But for now, it's fine.